src/main/java/seedu/address/commons/core/Alias.java

```diff
@@ -0,0 +1,82 @@
+//@@author briyanii-reused
```

Comment:
Is there a link to this source? Also, if you understood and adapted it, consider using
`//Solution below adapted from ...`

Reference: https://nus-cs2103-ay2021s2.github.io/website/admin/appendixB-policies.html#giving-credit-for-reused-work

Saw this in a few other files.